docs/user-guide/environment-variables.md

```diff
@@ -71,3 +76,21 @@ For example, given a REST API that provides access to different types of record,
 a subflow could be created to access the API and handle the response, using an
 environment variable to identify which type of record should be accessed. Individual
 instances of the Subflow can then be customised for those particular types.
+
+### Flow/Group level environment variables
+
+Since [version 2.1](https://nodered.org/blog/2021/10/21/version-2-1-released#flowgroup-level-environment-variables), environment variables can be set at the flow or group level. This is done in the appropriate tab in the edit dialog for the flow or group.
+
+### Raspberry Pi OS considerations
```

Comment:
While the script is mainly aimed at Pi users, it works well for any Debian based install eg Ubuntu. So this section may need renaming.